cmd/action/provider.go

```diff
@@ -22,10 +22,10 @@ func mysqlProvider(dsn string) (*Driver, error) {
 		return nil, err
 	}
 	return &Driver{
-		Driver:        drv,
-		MarshalSpec:   mysql.MarshalSpec,
-		UnmarshalSpec: mysql.UnmarshalSpec,
-		Types:         mysql.TypeRegistry.Specs(),
+		Driver:      drv,
+		Marshaler:   mysql.MarshalHCL, // TODO(rotemtam): change this when more syntaxes available.
```

Comment:
```suggestion
		Marshaler:   mysql.MarshalHCL,
```

Don't think this comment is needed.